src/fastly/vcl.js

```diff
@@ -26,6 +26,12 @@ async function init(fastly, version) {
   return fastly.setMainVCL(version, 'helix.vcl');
 }
 
+async function extensions(fastly, version) {
```

Comment:
How do you pass the customizations?

Reply:
This will be part of https://github.com/adobe/helix-publish/issues/79 (clear separation of code ;) )